tests/boot/test_mcuboot/testcase.yaml

```diff
@@ -12,14 +12,33 @@ common:
 tests:
   bootloader.mcuboot:
     tags: mcuboot
-    skip: true # disabled due to issue #58080
+    #skip: true # disabled due to issue #58080
     platform_allow:
       - frdm_k64f
       - mimxrt1060_evk
-      - nrf52840dk_nrf52840
```

Comment:
Do we want to disable nrf52840dk_nrf52840?

Reply:
Nordic guys add https://github.com/zephyrproject-rtos/zephyr/issues/58080 for their platform, so to enable us, I have to skip theirs for now